express/src/views/account/account.njk

```diff
@@ -41,15 +41,7 @@
               value: {
               text: emailAddress
             },
-              actions: {
-              items: [
-                {
-                  href: '/change-email-address',
-                  text: "Change",
-                  visuallyHiddenText: "email address"
-                }
-              ]
-            }
+              actions: {}
```

Comment:
The actions key can just be removed from the row.

Reply:
Fixed